samples/cpp/calibration.cpp

```diff
@@ -476,6 +476,20 @@ int main( int argc, char** argv )
     if ( boardSize.height <= 0 )
         return fprintf( stderr, "Invalid board height\n" ), -1;
 
+    cv::aruco::Dictionary dictionary;
+    if (dictFilename == "None") {
+        dictionary = aruco::getPredefinedDictionary(aruco::PredefinedDictionaryType(arucoDict));
+    }
+    else {
+        cv::FileStorage dict_file(dictFilename, cv::FileStorage::Mode::READ);
+        cv::FileNode fn(dict_file.root());
+        dictionary.readDictionary(fn);
+    }
+    cv::aruco::CharucoBoard ch_board({ boardSize.width + 1, boardSize.height + 1 },
```

Comment:
Why +1?

Reply:
Discussed the issue offline. Decided to remove +1, but add note about the board size units. Chessboard and radon board size are defined in inner corners, all other boards - in elements (circles or squares).